Dockerfile

```diff
@@ -1,10 +1,16 @@
-FROM openjdk:8-jdk-alpine
+FROM openjdk:8-jre-slim
 
-MAINTAINER Johannes Molland <johannes.molland@difi.no>
+RUN groupadd -o -g 1000 java \
+    && useradd -o -r -m -u 1000 -g 1000 java
 
-ARG jarPath
-ADD ${jarPath} app.jar
+ENV APP_DIR=/opt/deploymanager \
+    JAVA_OPTS=""
 
-ENV JAVA_OPTS=""
+ADD /target/*.jar ${APP_DIR}/app.jar
 
-ENTRYPOINT [ "sh", "-c", "exec java $JAVA_OPTS -Djava.security.egd=file:/dev/./urandom -jar /app.jar" ]
\ No newline at end of file
+RUN chown -R java:java ${APP_DIR}
+RUN chmod +x ${APP_DIR}/*
+
+USER java
+
+ENTRYPOINT [ "sh", "-c", "exec java $JAVA_OPTS -Djava.security.egd=file:/dev/./urandom -jar ${APP_DIR}/app.jar" ]
```

Comment:
Treng me ikkje nokre argument på slutten her, for å kunna gi inn parametrar?

Reply:
Godt poeng, kan legge til slik som det er gjort i de andre bildene.

Reply:
Hmm, jeg er litt usikker her nå. Virker som det er flere måter å gjøre dette på? Ser de filene jeg har endret på tidligere har hatt ", "@"]" i slutten. Men, vi kan vel også bruke -e NAVN="VERDI" for å sende parametre.

https://stackoverflow.com/questions/32727594/how-to-pass-arguments-to-shell-script-through-docker-run